src/src/PluginStructs/P099_data_struct.cpp

```diff
@@ -252,7 +269,7 @@ void P099_data_struct::scaleRawToCalibrated(uint16_t &x, uint16_t &y) {
       if (_x > StoredSettings.Calibration.bottom_right.x) {
         _x = StoredSettings.Calibration.bottom_right.x;
       }
-      float x_fact = (StoredSettings.Calibration.bottom_right.x - StoredSettings.Calibration.top_left.x) / ts_x_res;
+      float x_fact = (StoredSettings.Calibration.bottom_right.x - StoredSettings.Calibration.top_left.x) / _ts_x_res;
       x = int((_x * 1.0f) / x_fact);
     }
     uint16_t _y = y - StoredSettings.Calibration.top_left.y;

```

Comment:
`_y` is not a member here, but just a local variable.
So why using an underscore as prefix here?   (same for `_x`)

Reply:
This was written before we chose the underscore as prefix for member variables. Will fix.